plugins/postman/plugin.go

```diff
@@ -0,0 +1,22 @@
+package postman
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "postman",
+		Platform: schema.PlatformInfo{
+			Name:     "postman",
+			Homepage: sdk.URL("https://identity.getpostman.com/"), 
```

Comment:
To be consistent with other plugins, let's link this to the marketing homepage instead of the webapp itself.

```suggestion
			Homepage: sdk.URL("https://www.postman.com/"), 
```